debug-plugin.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
Worth putting this into `build/` or something vs. the top-level?

Reply:
It seems to belong in build, I agree. But I really like typing `./de`+`tab` and I don't want to have to type `./build/de`+`tab`. I'm lazy. :)

Reply:
I did add a `kill-plugin.sh` to the build directory. I run that far less frequently, so it definitely belongs there. 
We will remove `kill-plugin.sh` when this bug of plugins not being shut down is taken care of.

Reply:
Oh, somehow I had assumed this script was going to be executed from the context of a `make debug` target. Actually, that seems slightly nicer since you can assume the proper input arguments and don't need to invoke `manifest id` yourself. 2/5 on keeping all the targets in the `Makefile` vs. having other entry points

Reply:
Agree that this should be part of the Makefile.